src/language/custom-types/members.md

```diff
@@ -329,3 +329,22 @@ impl Point {
     pub fn set_y(self, val: i32) -> Self { Self::new(self.x, val) }
 }
 ```
+
+It is also possible to use the struct update syntax to achieve something similar. This moves the original value:
+
+```rust
+mod points {
+    #[derive(Debug)]
+    pub struct Point { pub x: i32, pub y: i32 }
+}
+
+fn main() {
+    use points::Point;
+    let pt = Point { x: 123, y: 456 };
+    let pt = Point { x: 789, ..pt };
+    println!("{pt:?}"); // prints: Point { x: 789, y: 456 }
+}
+```
+
+As seen in the example above, the struct update syntax requires access to the type fields.
+Therefore, it is generally more common to use it within the module that has access to private details of its types.
```

Comment:
Wrapping and adding a link to the [struct update syntax](https://doc.rust-lang.org/stable/book/ch05-01-defining-structs.html#creating-instances-from-other-instances-with-struct-update-syntax) bit in the Rust book:

```suggestion
As seen in the example above, the struct update syntax requires access to the
type's fields. Therefore, it is generally more common to see it used within
the module that has access to private details of its types.

  [struct update syntax]: https://doc.rust-lang.org/stable/book/ch05-01-defining-structs.html#creating-instances-from-other-instances-with-struct-update-syntax
```